flake.naersk

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
Do you need the nix stuff in here or should it be local only?
I don't mind having them in here if they help people but want to avoid unnecessary/conflicting files.
It also looks like a lot of the lines are commented.

Reply:
Oops, that's a mistake! That was some old experiment to compile Hacspec & examples with Nix...
Just rebased that.